package.json

```diff
@@ -15,7 +15,7 @@
   "license": "UNLICENSE",
   "dependencies": {
     "axios": "^0.21.2",
-    "discord.js": "git://github.com/discordjs/discord.js#cee6cf70ce76e9b06dc7f25bfd77498e18d7c8d4",
+    "discord.js": "^13.8.0",
```

Comment:
Thanks! Nice catch!

This change should have caused `package-lock.json` to update as well... did that happen on your system? If so, please add the package-lock change to the PR as well

Reply:
Ye uploaded that too